### PR TITLE
Allow change update channel via public API

### PR DESCRIPTION
**Pull request by VicDeo**
Ref https://github.com/owncloud/updater/issues/122 https://github.com/owncloud/updater/issues/126

### CODE REVIEW

**Comment by karlitschek** [comment]
as discussed :+1:

**Comment by VicDeo** [comment]
@karlitschek Is it crutucal for 8.1?

**Comment by karlitschek** [comment]
i think so actually

**Comment by VicDeo** [comment]
@karlitschek ok, setting a proper milestone

**Comment by PVince81** [comment]
Would be nice to have unit tests, if possible.
The logic is a bit more than just get/set, it seems to reset "version.php" loading too.

**Comment by VicDeo** [comment]
@PVince81 done

File: lib/public/util.php [hunk 1]
```diff
@@ -69,6 +69,24 @@ class Util {
 	public static function getVersion() {
 		return(\OC_Util::getVersion());
 	}
+	
+	/**
+	 * Set current update channel
+	 * @param string $channel
+	 */
+	public static function setChannel($channel) {
```
Comment (MorrisJobke) [inline lib/public/util.php]: cc @DeepDiver1975 regarding `\OCP\Util` stuff

**Comment by PVince81** [comment]
Makes sense :+1:

**Comment by VicDeo** [comment]
@DeepDiver1975 can you have a look?

**Comment by DeepDiver1975** [comment]
:+1:

**Pull request merged**